docs/1.5/extensions/external-links.md

```diff
@@ -124,22 +124,40 @@ class MyCustomLinkRenderer implements InlineRendererInterface
 
 ## Adding Icons
 
-You can also use CSS to automagically add an external link icon by targeting the `html_class` given in the configuration:
+You can also use CSS to add a custom icon by targeting the `html_class` given in the configuration:
 
-```css
-// Font Awesome example:
+```php
+$config = [
+    'external_link' => [
+        'html_class' => 'external',
+    ],
+];
+```
+
+```scss
```

Comment:
Not everyone uses SCSS (though I personally think they should!)  so should we change this example to work without needing to use `@extend`?  I'm open to your thoughts here.

Reply:
Let's just get rid of all the SCSS and fa/bootstrap examples.